fixtures/app_with_spm_dependencies/Tuist/Package.swift

```diff
@@ -6,10 +6,20 @@ import PackageDescription
     import ProjectDescriptionHelpers
 
     let packageSettings = PackageSettings(
+        productDestinations: [
+            "Styles": [
+                .iPad,
+                .iPhone,
+                .macCatalyst,
+                .macWithiPadDesign,
+                .appleVisionWithiPadDesign,
+            ],
```

Comment:
We should be able to remove [this](https://github.com/tuist/tuist/pull/6436/files#r1658446359) is resolved

Reply:
It has been removed.